src/AppBundle/Controller/SettingsController.php

```diff
@@ -20,4 +25,115 @@ public function indexAction()
             'client' => $client,
         ];
     }
-}
+
+    /**
+     * @Route("/deputyship-details/your-details/change-password", name="user_password_edit")
+     * @Route("/pa/settings/your-details/change-password", name="pa_profile_password_edit")
+     * @Template()
+     */
+    public function passwordEditAction(Request $request)
```

Comment:
git doesn't show the differences with the old action in the deleted controller, so difficult to understand what changed here